.github/ISSUE_TEMPLATE/usage_question.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Library-specific Usage Question
+about: Ask a question about this library that is neither a bug report nor feature request.
+title: ''
+labels: question, triage
+assignees: ''
+
+---
+
+<!-- PLEASE READ
+This form is for usage questions specific to this library.
+
+For questions that are related to the Google Ads API itself and not specific to this library, please reach out to one of our support channels: https://developers.google.com/google-ads/api/support.
+
+For troubleshooting tips, see: https://developers.google.com/google-ads/api/docs/best-practices/troubleshooting.
+
+Please check existing issues to see if your question has already been asked before creating a new issue. We encourage you to add any relevant information to the existing issue if one already exists.
+-->
+
+[DO NOT INCLUDE ANY PERSONAL OR SENSITIVE DATA - MAKE SURE TO REDACT CONTENT WHEN NECESSARY]
```

Comment:
Same here -- is this needed for usage questions? I expect it to be relevant mainly when logs are shared.

Reply:
Someone may use this instead of the Bug Report template for whatever reason, I think there is no harm adding it for safety especially since this template is fairly small and free-text.

Reply:
+1. Please retain this sentence.